pkg/qemu/qemu.go

```diff
@@ -30,29 +31,65 @@ var DefaultTimeout = 7 * time.Second
 // QEMU on a slow machine.
 var TimeoutMultiplier = 2.0
 
-// QEMU is filled and pass to `Start()`.
-type QEMU struct {
+type Network struct {
+	port   uint16
+	numVMs uint8
+}
+
+func NewNetwork() *Network {
+	return &Network{
+		port: 1234,
+	}
+}
+
+func (n *Network) newVM() *networkState {
+	num := n.numVMs
+	n.numVMs++
+	return &networkState{
+		connect: num != 0,
+		mac:     net.HardwareAddr{0x0e, 0x00, 0x00, 0x00, 0x00, byte(num)},
+		port:    n.port,
+	}
+}
+
+type networkState struct {
+	// Whether to connect or listen.
+	connect bool
+	mac     net.HardwareAddr
+	port    uint16
+}
+
+// Options is filled and pass to `Start()`.
+type Options struct {
 	// Path to the bzImage kernel
 	Kernel string
 
-	// Path to the initramfs
-	InitRAMFS string
+	// Path to the initramfs.
+	Initramfs string
```

Comment:
Is Go style to capitalize the acronyms?
```suggestion
	InitRAMFS string
```

Reply:
I think initramfs has become its own word at this point, which is why I changed it